app/Http/Requests/Api/UserRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests\Api;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UserRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'name' => 'required|string',
+            'email' => 'required|email',
+            'password' => 'required|string',
+            'confirmPassowrd' => 'required|string|same:password',
```

Comment:
also, the spelling of confirm password is incorrect.

Reply:
Now it has been removed. Because `confirmed` has been added to the password rule. Now according to password rules, we need to add one more parameter to confirm the password in the payload named `password_confirmation`. By which it will be known whether the password and confirm password match or not.